index.html

```diff
@@ -5,39 +5,167 @@
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-year"id="info-client">2015</li>
+            </ul>
+            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="language-used">
+              <div class="tags">html</div>
+              <div class="tags">css</div>
+              <div class="tags">javaScript</div>
+          </div>
+          <button type="button" class="button">See Project!</button>
+        </div>
+      </div>
 
-        <!-- Add font awesome icons -->
-        <ul class="icon-list">
-          <li><a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
-          <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
-          <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
-          <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
-          <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
-        </ul>
+      <!-- The "about me section is here"-->
+      <div class="about-me">
+        <h2> About Myself </h2>
+        <p> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="social">
+          <div class="let-connect">Let's Connect</div>
+          <ul class="icon-list">
+            <a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
+            <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
+            <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
+            <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
+            <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
+          </ul>
+        </div>
+        <button type="button" class="button">Get Resume</button>
+        <div class="self-category">
+          <hx>Languages</hx>
+            <ul class="dropdown">
+              <li>javaScript</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </nav>
+          <hx>Frameworks</hx>
+          <ul>
+            <li></li>
+            <li></li>
+            <li></li>
+          </ul>
+          <hx>Skills</hx>
+          <ul>
+            <li></li>
+            <li></li>
+            <li></li>
+          </ul>
+        </div>
       </div>
     </div>
   </body>
+  <footer>
+  </footer>
```

Comment:
[OPTIONAL] In this milestone we're still not adding a `footer` to our portfolio. But I just want to point out, that this element shouldn't be outside the `body` tag. That's because when the document renders in the browser, only the elements inside the `body` tag are rendered.